build.cmd

```diff
@@ -4,51 +4,96 @@ PUSHD %~dp0
 SETLOCAL
 SETLOCAL ENABLEDELAYEDEXPANSION
 
+:Loop
+if [%1]==[] GOTO Begin
+
+if /I [%1]==[Debug] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[Release] (
+    SET Configuration=%1
+    GOTO Next
+)
+
+if /I [%1]==[PROD] (
+    SET Environment=PROD
+    GOTO Next
+)
+
+if /I [%1]==[raw] (
+    SET UpdateTemplate=%1
```

Comment:
that's what I mean, so "build raw" shouldn't named with "UpdateTemplate"? how about "SkipTemplate"? Also update line 73 error msg

---
In reply to: [68714739](https://github.com/dotnet/docfx/pull/435#discussion_r68714739) [](ancestors = 68714739,68713739)